client/v2/autocli/msg.go

```diff
@@ -115,17 +115,19 @@ func (b *Builder) BuildMsgMethodCommand(descriptor protoreflect.MethodDescriptor
 		}
 
 		// enable sign mode textual and config tx options
-		b.TxConfigOpts.EnabledSignModes = append(b.TxConfigOpts.EnabledSignModes, signing.SignMode_SIGN_MODE_TEXTUAL)
-		b.TxConfigOpts.TextualCoinMetadataQueryFn = authtxconfig.NewGRPCCoinMetadataQueryFn(clientCtx)
+		if !clientCtx.Offline {
```

Comment:
Mmmh, that does make it re-add wire it for depinject enabled app.
Imho, we want an extra check like `if b.TxConfigOpts.EnabledSignModes not contains signing.SignMode_SIGN_MODE_TEXTUAL`

Reply:
Replaced it with `if !clientCtx.Offline && !slices.Contains(b.TxConfigOpts.EnabledSignModes, signing.SignMode_SIGN_MODE_TEXTUAL) {`